### PR TITLE
reformat and rearrange code

### PR DESCRIPTION
**Pull request by noahschoenthal**
Code gets rearranged and a cleanup

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as environment from '../config/environment.json';
-import {PLATFORM} from 'aurelia-pal';
-import {I18N, TCustomAttribute} from 'aurelia-i18n';
+import { PLATFORM } from 'aurelia-pal';
+import { I18N, TCustomAttribute } from 'aurelia-i18n';
```
Comment (noahschoenthal) [inline src/main.js:3]: @luca-bruegger Please consider writing spaces between those curly braces.

Comment (luca-bruegger) [reply src/main.js:3]: Sure. I will.

File: src/app.css [hunk 2]
```diff
@@ -146,7 +143,7 @@ section {
   font-family: 'Ruda', sans-serif;
 }
 
-.footer > div {
+.footer>div {
```
Comment (luca-bruegger) [inline src/app.css]: @noahschoenthal please undo this. Looks a lot cleaner to me, with a space in between.

File: src/app.css [hunk 3]
```diff
@@ -227,7 +222,7 @@ h1 {
   align-content: center;
 }
 
-.download-button > a {
+.download-button>a {
```
Comment (luca-bruegger) [inline src/app.css]: Here aswell

File: src/app.css [hunk 4]
```diff
@@ -246,11 +241,11 @@ a {
   width: 350px;
 }
 
-.download-button > a > svg:before {
+.download-button>a>svg:before {
```
Comment (luca-bruegger) [inline src/app.css]: Same here. Please fix it everywhere.

File: src/app.css [hunk 5]
```diff
@@ -342,8 +336,10 @@ a {
   }
 }
 
-@-webkit-keyframes arrow /*Safari and Chrome*/
-{
+@-webkit-keyframes arrow
+/*Safari and Chrome*/
+
```
Comment (luca-bruegger) [inline src/app.css]: Good point. But move the comment above the css selector.

**Review by luca-bruegger — CHANGES_REQUESTED** [review]
PR is on a good path. Still some things u need to fix 🚀

File: src/app.css (hunk 5 above)
Comment (noahschoenthal) [reply src/app.css]: @luca-bruegger Look at the newly pushed comment does it meet your expectations?

**Review by luca-bruegger — APPROVED** [review]

**Pull request merged**